exporter/prometheusexporter/collector.go

```diff
@@ -100,46 +96,18 @@ func (c *collector) getMetricMetadata(metric pdata.Metric, labels pdata.StringMa
 	), values
 }
 
-func (c *collector) convertIntGauge(metric pdata.Metric) (prometheus.Metric, error) {
-	ip := metric.IntGauge().DataPoints().At(0)
-
-	desc, labels := c.getMetricMetadata(metric, ip.LabelsMap())
-	m, err := prometheus.NewConstMetric(desc, prometheus.GaugeValue, float64(ip.Value()), labels...)
-	if err != nil {
-		return nil, err
-	}
-
-	if c.sendTimestamps {
-		return prometheus.NewMetricWithTimestamp(ip.Timestamp().AsTime(), m), nil
-	}
-	return m, nil
-}
-
 func (c *collector) convertDoubleGauge(metric pdata.Metric) (prometheus.Metric, error) {
```

Comment:
```suggestion
func (c *collector) convertGauge(metric pdata.Metric) (prometheus.Metric, error) {
```

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/3715 this was in the other PR